plugins/abnormal_security/plugin.spec.yaml

```diff
@@ -250,13 +250,13 @@ actions:
         description: This input enables you to filter your results from a certain date, the date has to be in ISO 8601 format - YYYY-MM-DDTHH:MM:SSZ
         type: string
         required: false
-        example: 2021-03-01T21:11:38Z
+        example: 2021-03-11 21:11:38
```

Comment:
This is not an ISO format described in the description `YYYY-MM-DDTHH:MM:SSZ`. Kindly please roll back to the previous examples, and fill out the help.md file with proper ones 😄